x-pack/legacy/plugins/apm/scripts/kibana-security/setup-custom-kibana-user-role.ts

```diff
@@ -20,13 +20,13 @@ const config = yaml.safeLoad(
   )
 );
 
-const GITHUB_USERNAME = argv.username as string;
 const KIBANA_INDEX = config['kibana.index'] as string;
 const TASK_MANAGER_INDEX = config['xpack.task_manager.index'] as string;
-const ELASTICSEARCH_USERNAME = (argv.esUsername as string) || 'elastic';
-const ELASTICSEARCH_PASSWORD = (argv.esPassword ||
+const KIBANA_ROLE_SUFFIX = argv.roleSuffix as string;
```

Comment:
Why `const KIBANA_ROLE_SUFFIX = argv.roleSuffix as string;` and not `const KIBANA_ROLE_SUFFIX: string = argv.roleSuffix;`?

Reply:
`arg.roleSuffix` is an `unknown`

![image](https://user-images.githubusercontent.com/209966/71036757-945bc880-211e-11ea-8443-25c045fd390d.png)